src/main/resources/META-INF/resources/admin.xhtml

```diff
@@ -71,10 +71,13 @@
             </aside>
             <!-- content -->
             <div id="content" class="content-wrapper">
+                <adm:breadcrumb title="#{adminConfig.autoRenderBreadCrumbTitle ? title : ''}"/>
                 <ui:insert name="content-wrapper"/>
                 <section class="content-header">
                     <h1>
-                        <ui:insert name="title"/>
+                        <ui:insert name="title">
+                            <h:outputText value="#{title}" rendered="#{not empty title}" />
+                        </ui:insert>
```

Comment:
I think we could use just:
```
 <adm:breadcrumb title="#{title}" rendered="#{not empty title}" /> <!-- the rendered should work, see here https://stackoverflow.com/questions/8106410/why-does-composite-component-rendered-attribute-throw-an-illegalargument-excep-->
 <ui:insert name="title">
              <h:outputText value="#{title}" rendered="#{not empty title}" />
 </ui:insert>
```

This way if user wants to use the "automatic" breadCrumb it just needs to declare the title ui:param.

On the other hand if user wants a conditional title on the breadCrumb (e.g : [like here](https://github.com/adminfaces/admin-starter/blob/master/src/main/webapp/car-form.xhtml#L18)) it can just use the current approach of declaring the breadCrumb on the page.  

What do you think?

Reply:
This is exactly what I did first. But then the `add` method of the listener inside `<f:event type="preRenderView">` of the `breadcrumb.xhtml` was always called, as if the `rendered` attribute of the composite component did not have any effect. 

I know it should work, as the `rendered` attribute is inherited by `UIComponentBase`, but it did not. I even tried a separate `<ui:fragment rendered="#{cc.attrs.rendered}">` inside the cc, but also no luck :thinking:

Reply:
I'll make some tests on your branch asap.

Reply:
Yea, pre render view is fired even when rendered is false...

The workaround is to skip pre render view if rendered attr is false (no title is provided):

```
 <adm:breadcrumb title="#{title}" rendered="#{not empty title}"/>
```

Then on the composite add an extra parameter to the preRenderView method:

```
        <f:event type="preRenderView" listener="#{breadCrumbMB.add(cc.attrs.link, cc.attrs.title, cc.attrs.clear, cc.rendered)}"/>
```

Finally on the BreadCrumbMB just add the breadCrumb item if rendered is true (param 'title' is present on page):

```
public void add(String link, String title, Boolean clear, Boolean shouldAdd) {
        if(shouldAdd) {
            this.add(link, title, clear);
        }
    }

```

Makes sense?

Reply:
Yeah, makes sense, and works perfectly. I pushed a new version without the configuration just now.